src/Pages/Shared/LinkedCIDetailsModal/LinkedCIDetailsModal.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import {
+    Drawer,
+    InfoColourBar,
+    Pagination,
+    SearchBar,
+    SortableTableHeaderCell,
+    SortingOrder,
+    getCommonSelectStyle,
+} from '@devtron-labs/devtron-fe-common-lib'
+import ReactSelect from 'react-select'
+import { LinkedCIDetailsModalProps } from './types'
+import { ReactComponent as Info } from '../../../assets/icons/ic-info-filled.svg'
+import { ReactComponent as Close } from '../../../assets/icons/ic-close.svg'
+
+import AppListData from './constants'
+
+const commonStyles = getCommonSelectStyle()
+
+export interface AppListRowDataType {
+    appName: string
+    environment: string
+    triggerMode: string
+    deploymentStatus: string
+}
+
+const AppListRow = ({ appName, environment, triggerMode, deploymentStatus }: AppListRowDataType) => {
+    return (
+        <div className="flexbox pl-20 pr-20 pt-8 pb-8 dc__align-self-stretch dc__align-center dc__gap-16 fs-13 fw-4 dc__content-space">
+            <span>{appName}</span>
+            <span>{environment}</span>
+            <span>{triggerMode}</span>
+            <span> {deploymentStatus} </span>
+        </div>
+    )
+}
+
+const AppList = () => {
+    return (
+        <div className="">
+            <div className="flexbox bcn-0 dc__uppercase pl-20 pr-20 pt-6 pb-6 dc__border-bottom dc__align-self-stretch dc__align-center dc__gap-16 fs-12 fw-6 dc__content-space">
```

Comment:
Group similar classes, add line height. We can ditch the padding blocks and use grid-auto-row to define row height
```suggestion
            <div className="flex dc__gap-16 bcn-0 dc__uppercase pl-20 pr-20 pt-6 pb-6 dc__border-bottom fs-12 fw-6 dc__content-space">
```

Reply:
The layout would not have any relationship b/w rows. PS: use grid